reflex/event.py

```diff
@@ -767,17 +767,17 @@ def set_focus(ref: str) -> EventSpec:
     )
 
 
-def scroll_to(elem_id: str) -> EventSpec:
+def scroll_to(elem_id: str, align_to_top: bool | Var[bool] = True) -> EventSpec:
     """Select the id of a html element for scrolling into view.
 
     Args:
-        elem_id: the id of the element
+        elem_id: The id of the element to scroll to.
+        align_to_top: Whether to scroll to the top (True) or bottom (False) of the element.
 
     Returns:
         An EventSpec to scroll the page to the selected element.
     """
-    js_code = f"document.getElementById('{elem_id}').scrollIntoView();"
-
+    js_code = f"document.getElementById('{elem_id}').scrollIntoView({str(Var.create(align_to_top))});"
```

Comment:
`str` casting this Var will cause it to lose any `VarData`, so if someone inlines `rx.scroll_to` with `align_to_top` as a state Var, then it might throw a frontend error.

if they call it from the backend, it's fine, because they will be passing a real `bool` from the state instance, rather than a Var

Reply:
Tested this out with @adhami3310 's change and it works on my app